.github/workflows/verify.yaml

```diff
@@ -340,6 +340,32 @@ jobs:
         working-directory: python-sdk
         run: make test
 
+  build-verify-go-sdk:
+    name: Verify Go SDK
+    runs-on: ubuntu-20.04
+    # if: github.repository_owner == 'Apicurio' && !contains(github.event.*.labels.*.name, 'DO NOT MERGE')
+    steps:
+      - name: Checkout Code with Ref '${{ github.ref }}'
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
+          cache: 'maven'
+
+      - name: Go - Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.20'
+
+      - name: Build Registry
```

Comment:
We must start considering having a previous step in building the project instead of building it in every single job (not saying it must be done in this PR, just making it clear).

Reply:
https://github.com/Apicurio/apicurio-registry/issues/4137

Reply:
agree, that will be a nice optimization.